azerothcore/azerothcore-wotlk
# fix(Script/Quest): Moved Shay Leafrunner's script from DB to C++.

Fixes #1535
Fixes #6722



## Issues Addressed:

- Closes #1535
- Closes #6722
- Closes https://github.com/chromiecraft/chromiecraft/issues/1083

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

1. Teleport to GUID 50144
2. Accept the quest Wandering Shay and loot the bell from the box nea her feet
3. Escort her to Rockbiter camp and see it does complete the quest
4. Observe the whole script.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
